README.md

```diff
@@ -192,3 +192,26 @@ root@ad84f5154816:/#
   <summary>What's with the dellar name?</summary>
     These commands draw inspiration from [Stellar](https://pypi.org/project/stellar/) — a database snapshot and restore tool that is no longer maintained.
 </details>
+
+## Tests
+
+Run python unit tests in the `web` container as follows:
+
+```sh
+# Run the whole test suite once.
+djtest
+
+# Run the whole test suite, collecting test coverage information.
+djcov
+```
+
+JavaScript unit tests for this project use [Jest](https://jestjs.io/). Here are commands you can use:
+
+```sh
```

Comment:
I believe there's already documentation on running javascript tests in docs/frontend. I wonder if the back-end tests should also live in the docs rather than the readme?